core/tag.c

```diff
@@ -124,6 +124,26 @@ tag_t lf_delay_strict(tag_t tag, interval_t interval) {
   return result;
 }
 
+/**
+ * @brief Return the greatest tag earlier than the given tag.
+ *
+ * If the given tag is `FOREVER_TAG` or `NEVER_TAG`, however, just return the given tag.
+ * @param tag The tag.
+ */
```

Comment:
I suggest removing the function comment from the .c file. It's in the .h file, and it's better to not have copies that will inevitably become inconsistent.

Reply:
Resolved in [896e152](https://github.com/lf-lang/reactor-c/pull/349/commits/896e15293bd21958a124d7f73b02c455340f81d7).